test/jenkins.parsetestargs.cmd

```diff
@@ -62,6 +62,11 @@ if not "%1"=="" (
         REM fallthrough to default (also add this to %_ExtraTestArgs%)

```

Comment:
The fallthrough is to default.  If %1 matches -disablejit, it won't match -lite. So they are already mutually exclusive so that it will be added to _ExtraTestArgs